ci-operator/config/openshift/dptp-workflow-test/openshift-dptp-workflow-test-master.yaml

```diff
@@ -30,7 +30,7 @@ tests:
     cluster_profile: osd-ephemeral
     env:
       CLUSTER_DURATION: "300"
-      CLUSTER_VERSION: 4.6.12
+      CLUSTER_VERSION: 4.10.3
```

Comment:
I would say if you're going to bump the CLUSTER_VERSION, then you should also bump the other `4.6` values up to `4.10` as well.

Reply:
When I tried it previously (to update all occurrences of 4.6 to 4.10), the check `
ci/prow/prow-config-filenames` failed.
But now, I have tried again and the same check passed. So, I will keep it.